pyspecdata/core.py

```diff
@@ -3321,8 +3333,6 @@ def set_error(self,*args):
             self.data_error = reshape(array(args[0]),shape(self.data))
         elif (len(args) is 2) and (isinstance(args[0], str)) and (isinstance(args[1], ndarray)):
             self.axis_coords_error[self.axn(args[0])] = args[1]
-        elif (len(args) is 2) and (isinstance(args[0], str)) and (isscalar(args[1])):
-            self.axis_coords_error[self.axn(args[0])] = args[1]*ones_like(self.getaxis(args[0]))
```

Comment:
this was added by me, please leave

Reply:
OK, sorry, I got confused, and thought you had already reverted the bad commit, but this was part of it